### DISCUSSION TITLE
Failed to apply for faculty benefits: "Please complete your GitHub billing information with your full name"

### DISCUSSION THREAD

**Discussion by nneonneo** [community/community#118746] [GitHub Education]
### Hi everyone,

My GitHub faculty benefits expired, so I am renewing the benefits. However, this new automated system is not working for me at all.

I have tried several times to upload or photograph documents now, but every time I see the following:

> Please complete your [GitHub billing information](https://github.com/settings/billing/payment_information) with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying. If you have only a single legal name, enter it in both the first and last name fields.

I have tried logging out and back in, switching browsers, using private mode. I have tried uploading scans of my ID card and scans of my employment verification letter from the University. I have tried photographing my ID card and the official employment verification letter. I have tried using the camera on my phone, as well as the camera on my laptop. I have tried using my full legal name (as it appears on the employment verification letter), as well as my preferred name (as it appears on my ID card), for both types of "proof". I have checked again and again that my billing information is correct and that the name corresponds with the document I am uploading. In every single case, I get the error as shown above. (I additionally get another error telling me to use my camera when I upload a document; if that's an error, why even offer uploads in the first place?).

As an additional issue, my ID card shows only the name, affiliation (Faculty) and employee ID on the front; the back has the expiry date but no name. I tried uploading a combined picture that shows the front and back, but the system doesn't even process it - it just loads me back to the upload proof page without any error message. The file is 1280 × 1659 pixels, 830 KB in JPEG format, which should perfectly fit the requirements, so I have no idea what's going on with that. But, in any case, the system won't even take my official employment verification letter (by upload or by photo), so I am completely out of options here.

**Comment by samus-aran** [comment]
Hi @nneonneo 👋 

Can you ensure the billing information is entered and saved on your profile before re-applying? This I suspect is is the biggest hurdle to overcome. The use of ID should be suitable so lets tackle the billing info first. 👍

**Reply by nneonneo** [reply]
As I noted in my question:

> I have checked again and again that my billing information is correct and that the name corresponds with the document I am uploading.

I have just checked yet again, and the information listed under "Billing information" in "Billing and plans -> Payment information" is correct. I have not added a payment method as the site clearly indicates that one is not necessary. However, my billing information is there, and it is definitely saved.
